.github/workflows/web-test.yml

```diff
@@ -9,6 +9,10 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        codepath: [web-admin, web-local]
```

Comment:
Shouldn't `web-common` also be in here? In case we add tests there

Reply:
web-common doesn't have same tests, few stages failed for it.